gradle.properties

```diff
@@ -21,6 +21,8 @@ group=org.apache.kafka
 #  - tests/kafkatest/version.py (variable DEV_VERSION)
 #  - kafka-merge-pr.py
 version=2.3.0-SNAPSHOT
+# Version used by CP subprojects that are not in AK
+cpVersion=5.3.0-SNAPSHOT
```

Comment:
Will this be covered by the normal CP version bump process, or will we have to remember to do it manually? It would be nice if we could get it from the common POM but I don't know of any way to do that.

Reply:
We will have to remember to do it manually.

Reply:
if it's not covered already, you should file a tools ticket to help with this. if we have to do this manually, it needs to be added to the release process documentation. but it should really be automated since much of the version bumping in release is just regex match and replace

Reply:
@ewencp I opened https://confluentinc.atlassian.net/projects/ST/issues/ST-1594